k8sdeps/transformer/inventory/inventory.go

```diff
@@ -16,24 +16,35 @@ import (
 	"sigs.k8s.io/kustomize/pkg/types"
 )
 
+type GarbagePolicy int
+
+const (
+	GarbageUnknown GarbagePolicy = iota
+	GarbageIgnore
+	GarbageCollect
+)
+
```

Comment:
You can define `GarbagePolicy` in `sigs.k8s.io/kustomize/pkg/types`. Then it won't introduce dependency on k8sdeps.

Reply:
good catch.  done!